dirsrvtests/tests/suites/logging/logconv_test.py

```diff
@@ -0,0 +1,980 @@
+        18. Test LDAPS connection stats
+        19. Test STARTTLS operation stats
+        20. Test behavior with missing access log
+        21. Test behavior with invalid access log format
+    :expectedresults:
+        1. Success
+        2. Success
+        3. Success
+        4. Success
+        5. Success
+        6. Success
+        7. Success
+        8. Success
+        9. Success
+        10. Success
+        11. Success
+        12. Success
+        13. Success
+        14. Success
+        15. Success
+        16. Success
+        17. Success
+        18. Success
+        19. Success
+        20. Success
+        21. Success
+    """
+    log.info(f"Running logconv test suite in {log_format.upper()} mode\n")
+
+    test_env = LogConvTest(fresh_instance, log_format)
+    failures = []
+
+    def run(test_func):
+        try:
+            test_func(test_env)
+            err = test_env.wait_for_log_counts(expected=test_env.expected, test_name=test_func.__name__)
+            if err:
+                full_name = f"{log_format} - {test_func.__name__}"
+                # raise AssertionError(f"\n\n=== Logconv Test Failure ===\n\n{full_name}:\n{err}")
+                failures.append((full_name, err))
+        except Exception as e:
+            full_name = f"{log_format} - {test_func.__name__}"
+            failures.append((full_name, f"Unexpected exception: {e}"))
+
+    test_cases = [
+        _test_add_del,
+        _test_modify,
+        _test_modrdn,
+        _test_compare,
+        _test_search,
+        _test_internal_ops,
+        _test_base_search,
+        _test_paged_search,
+        _test_persistant_search,
+        _test_simple_bind,
+        _test_anon_bind,
+        _test_ldapi,
+        _test_restart,
+        _test_invalid_filter,
+        _test_partially_unindexed_vlv_sort,
+        _test_fully_unindexed_search,
+        _test_ldaps,
+        _test_starttls,
+        _test_missing_access_log,
+        _test_invalid_access_log_format
+    ]
+
+    test_env.configure_logs(log_format)
+
+    for test in test_cases:
+        run(test)
+
+    if os.path.exists(test_env.debug_output):
+        os.remove(test_env.debug_output)
+
+    if failures:
+        summary_lines = []
+        summary_lines.append("=== Logconv Test Failures ===\n")
+
+        for name, error in failures:
+            summary_lines.append(f"{name}:")
+            summary_lines.extend(f"  {line}" for line in error.strip().splitlines())
+            summary_lines.append("")
+
+        summary = "\n".join(summary_lines)
+        raise AssertionError(f"\n{summary}")
+
+def _test_add_del(test_env):
+
+    def _add_del_users(test_env):
+        users = UserAccounts(test_env.inst, DEFAULT_SUFFIX)
+        try:
+            for idx in range(3):
+                users.create_test_user(uid=str(idx), gid=str(idx))
+        except ldap.ALREADY_EXISTS as e:
+            log.warning(f"_test_add_del - User already exists: {e}")
```

Comment:
Do we really need it?
I think we expect the test to be isolated...
But it's a minor issue, I don't have a strong preference.